Modules/core/Controller/CoreaccountController.php

```diff
@@ -111,6 +111,14 @@ public function indexAction() {
         $modelSpaces = new CoreSpace();
         $spaces = $modelSpaces->getForList();
 
+        // add structures names to spaces displayed names
+        for($i=0; $i<count($spaces['ids']); $i++) {
```

Comment:
not efficient: select from select from select and in a loop....
why not getting all menus and matching space id with structure name

Reply:
yes, that's why I thought about just after having pushed. There are obviously too many sql request here. I'm gonna fix that